docs/excel/custom-functions-runtime.md

```diff
@@ -0,0 +1,149 @@
+---
+ms.date: 09/18/2018
+description: Excel custom functions use a new JavaScript runtime, which differs from the standard Add-ins WebView control runtime.
+title: Runtime for Excel Custom Function Add-ins
+---
+
+# Runtime for Excel custom function add-ins
+
+Custom functions extend Excel’s capabilities using a new JavaScript runtime. This runtime utilizes a sandboxed JavaScript engine rather than a web browser. Additionally, it prioritizes performance, allowing you to run thousands of custom functions simultaneously.  
+
+> [NOTE!] The code for your add-in may include other parts, such as task panes and other UI elements.  
+> These will continue to run in the browser-like WebView runtime that you are used to. The new runtime only applies to the custom functions related code in your add-in.  
+
+## Notable changes
+
+- The new JavaScript runtime used by custom functions does not provide access to the Document Object Model (DOM) or support libraries like JQuery which rely on the DOM.
+- In the JavaScript file which defines your functions (if using yo office, **customfunctions.js**), you can now return a regular JavaScript `Promise` instead of using `OfficeExtension.Promise`.
+- In the JSON file which describes your functions (if using yo office, **customfunctions.json**), you no longer need to specify “sync” or “async” under “options”.  
```

Comment:
I'm not sure that "yo office" is the exact term and/or capitalization we use elsewhere in the docs. Suggest that you do a global search of our docs to see what term we're using elsewhere in the doc set, and update your custom functions docs to use the same term.

Reply:
Sure thing. Did a search and it doesn't seem like we actually call it by name very often or advise people to use it. So, I'm going to go with what we called in the Hands on Lab just to be consistent with something ("the Yo Office Yeoman generator").